src/components/SingleAutocompleteSelectField/SingleAutocompleteSelectField.tsx

```diff
@@ -41,7 +40,7 @@ export interface SingleAutocompleteSelectFieldProps extends Partial<FetchMorePro
   fetchChoices?: (value: string) => void;
   onChange: (event: React.ChangeEvent<any>) => void;
   fetchOnFocus?: boolean;
-  FormHelperTextProps?: ExtendedFormHelperTextProps;
+  FormHelperTextProps?: any;
```

Comment:
issue: please type it, at least some minimal type, but not any - adding any here we are adding more job for us later on when removing ignore comment